packages/autocomplete-preset-algolia/src/highlight/__tests__/reverseHighlightedParts.test.ts

```diff
@@ -0,0 +1,37 @@
+import { reverseHighlightedParts } from '../reverseHighlightedParts';
+
+describe('reverseHighlightedParts', () => {
+  test('returns a reversed partially highlighted parts array', () => {
+    expect(
+      reverseHighlightedParts([
+        { isHighlighted: true, value: 'amazon ((fire' },
+        { isHighlighted: false, value: 'tv)) tablet??' },
+      ])
+    ).toMatchObject([
```

Comment:
Any reasons for using `toMatchObject` over `toEqual`?

Reply:
It seems that there's no differences between both on a fully matching object. The naming of `toMatchObject` is more explicit IMHO but as we use `toEqual` in other tests, I should maybe change it.

Reply:
Yes, `toEqual` is more strict so we favor it.